Registry/settings.multiplayersample_gamelauncher.setreg

```diff
@@ -0,0 +1,11 @@
+{
+    "Amazon": {
+        "AzCore": {
+            "Runtime": {
+                "ConsoleCommands": {
+                    "loadlevel": "startmenu"
```

Comment:
Does this apply to Server Launcher?

Reply:
Nay, this PR only adds to __gamelauncher.setreg_

Suppose we could also add this to server launchers... I'd want to make sure loading a level in setreg for servers doesn't override editor-server does when it launches its server. Last I checked, setting the server setreg loadlevel _would_ load that lever, and then switch over to the original level the editor was attempting to test. But ideally we'd want to a way to suppress that setreg loadlevel when the editor runs CTRL+G.